README.md

```diff
@@ -1,40 +1,198 @@
-# Vault Integration Pack
-
-This pack is for Hashicorp Vault integrations
+# vault integration pack
```

Comment:
Worth capitalizing the title, as before
```suggestion
# Vault Integration Pack
```

Reply:
Capitalisation applied to jinja template and markdown.